index.d.ts

```diff
@@ -0,0 +1,331 @@
+import React from 'react';
+import { ViewStyle, TextStyle } from 'react-native';
+
+export interface IOption {
+  key: any;
+  label?: string;
+  section?: boolean;
+  accessibilityLabel?: string;
+  component?: React.ReactNode;
+  [key:string]: any;
+}
+
+type AnimationType = 'none' | 'slide' | 'fade';
+type OrientationType = 'portrait' | 'portrait-upside-down' | 'landscape' | 'landscape-left' | 'landscape-right';
+
+interface IModalSelectorProps<TOption> {
+  /**
+   * Array of objects with a unique key and label to select in the modal
+   * Optional component overrides label text
+   *
+   * Default is []
+   */
+  data: TOption[],
+  
+  /**
+   * Callback function, when the users has selected an option
+   *
+   * Default is () => {}
```

Comment:
Don't know if this is necessary, but could be nice to markdown this as ``Default is `() => {}` ``

Reply:
Yes, that's nice. I'll add it